frontend/src/metabase/browse/components/ModelsTable.tsx

```diff
@@ -81,11 +82,9 @@ export const ModelsTable = ({
   models = [],
   skeleton = false,
 }: ModelsTableProps) => {
-  // for large datasets, we need to simplify the display to avoid performance issues
   const isLargeDataset = models.length > LARGE_DATASET_THRESHOLD;
 
-  const [showLoadingManyRows, setShowLoadingManyRows] =
-    useState(isLargeDataset);
+  const [showLoadingManyRows, setShowLoadingManyRows] = useState(false);
```

Comment:
We may not need this anymore either

Reply:
I'm not seeing it even on a large dataset.  the skeletons are much nicer. go ahead and remove it.